File: Python/0051-0100/80. Remove Duplicates from Sorted Array II.py
# https://leetcode.com/problems/remove-duplicates-from-sorted-array-ii/

class Solution:
    def removeDuplicates(self, nums: List[int]) -> int:
        first = 1 # Two Pointers

        for second in range(1,len(nums)):
            if(first == 1 or nums[second] != nums[first-2]):
                nums[first] = nums[second]
                first += 1

        return first

File: Javascript/0051-0100/80. Remove Duplicates from Sorted Array II.js
// https://leetcode.com/problems/remove-duplicates-from-sorted-array-ii/

var removeDuplicates = function(nums) {
    let first = 1, second = 1; // Two Pointers

    for(;second<nums.length; second++)
        if(first == 1 || nums[second] != nums[first-2])
            nums[first++] = nums[second];

    return first;
};
